examples/plot_model_card.py

```diff
@@ -105,4 +99,12 @@
     model=pkl_name, requirements=[f"scikit-learn={sklearn.__version__}"], dst=local_repo
 )
 
-model_card.save(os.path.join(f"{local_repo}", "README.md"))
+model_card.save(
+    os.path.join(f"{local_repo}", "README.md"),
+    card_data=card_data,
+    limitations=limitations,
+    model_description=model_description,
+    citation_bibtex=citation,
+    model_card_authors=model_card_authors,
+    get_started_code=get_started_code,
```

Comment:
do we want to have two ways people can set these values? I think it'd be cleaner if the user would only have the `add` method to add these sections (or access through attributes) instead of having them in the `save`'s signature. WDYT @BenjaminBossan ?

Reply:
I refactored it accordingly, can you check? it enables method cascading like you asked.